resources/install.rb

```diff
@@ -43,22 +43,30 @@ def fetch_checksum
     # preserve the legacy name of sha1_base_path
     new_resource.checksum_base_path = new_resource.sha1_base_path if new_resource.sha1_base_path
     uri = if new_resource.tarball_uri.nil?
-            URI.join(new_resource.checksum_base_path, "tomcat-#{major_version}/v#{new_resource.version}/bin/apache-tomcat-#{new_resource.version}.tar.gz.md5")
+            unless new_resource.tarball_base_path.start_with?('file')
```

Comment:
Sure no problem. I'm aware it looked a little clunky so happy to make any corrections.